pages/ucla.tsx

```diff
@@ -0,0 +1,31 @@
+import { NextSeo } from 'next-seo';
+import React from 'react';
+import Layout from '../components/Layout';
+
+function UCLA(): JSX.Element {
+  return(
+    <Layout>
+      <div className='container'>
+        <NextSeo
+          title="contribute | open source at ACM at UCLA"
+          description="get started by contributing to one of our projects"
+          openGraph={{
+            images: [{
+              url: 'https://opensource.uclaacm.com/logo.png',
+              width: 1200,
+              height: 1200,
+              alt: 'The ACM at UCLA logo',
+            }],
+            site_name: 'open source at ACM at UCLA',
+          }}
+        />
+        <h1>have you wanted to work on projects used by ucla students??
```

Comment:
just a nitpick, but it's normally common practice for h1's to be relatively succinct and you can write a description about the page inside a p tag or an h6! Like i'd recommend restructuring it like so:

```html
<h1> ucla opensource</h1>
<h4>opensource projects by all ucla students. any project with the 
ucla-opensource tag will be available to work on.</h4>
<h6>this page is currently a work in progress</h6>
```

Reply:
Just a tidbit: that example would break the semantic meaning of the header tags (i.e. they should be descending; you should use CSS for styling instead of relying on HTML tags and their defaults). I do agree that a more succint header would be a good idea!